src/Worker/UpdateGServers.php

```diff
@@ -47,7 +48,7 @@ public static function execute()
 		}
 
 		$total = DBA::count('gserver');
-		$condition = ["`next_contact` < UTC_TIMESTAMP() AND (`nurl` != ? OR `url` != ?)", '', ''];
+		$condition = ["`next_contact` < ? AND (`nurl` OR `url`)", DateTimeFormat::utcNow()];
```

Comment:
I don't think that this query works. I just tested this:
```
> select nurl from gserver where not nurl limit 1;
+-----------------+
| nurl            |
+-----------------+
| http://snarl.de |
+-----------------+
1 row in set, 1 warning (0.00 sec)
```

Reply:
You're right, I wanted it to work but it simply doesn't.